android/app/src/main/java/com/apifm/MainApplication.java

```diff
@@ -21,7 +23,13 @@ public boolean getUseDeveloperSupport() {
           return BuildConfig.DEBUG;
         }
 
-        @Override
+        /* @Override
```

Comment:
Remove the commented out part and the imports since it doesn't look like anything is anything being changed here.